### ISSUE TITLE
Pulseeffects only launch from the terminal

### ISSUE THREAD

**Issue by francoism90** [wwmm/pulseeffects#865]
When I open Pulseeffects as application, nothing happens. Running the same command `pulseeffects` in a terminal, opens the GUI and everything seems to be working:
```
$ pulseeffects

(pulseeffects:2514198): pulseeffects-WARNING **: 16:18:08.316: rnnoise plugin was not found!

(pulseeffects:2514198): pulseeffects-CRITICAL **: 16:18:08.325: pulse_manager:  failed to get source info:

(pulseeffects:2514198): pulseeffects-WARNING **: 16:18:08.342: rnnoise plugin was not found!
```

This also prevents Pulseeffects from running on startup. I'm on Arch Linux and KDE. Is this a known issue? Any workaround? Thanks!

**Comment by wwmm** [comment]
The `rnnoise` plugin is optional. You can ignore this error. Check if pacman installed `pipewire-pulse` after an upgrade. The pulseeffects package available in the stable repository works only with Pulseaudio.

**Comment by wwmm** [comment]
If that is not the case then kill the current instance `pulseeffects -q` and restart it in debug mode `G_MESSAGES_DEBUG=pulseeffects pulseeffects` so we can see what is going on.

**Comment by francoism90** [comment]
@wwmm Best wishes for 2021. :)

It seems installing `pipewire-pulse` replaces Pulseaudio, is that correct? I'm on KDE and I don't think they do support Pipewire (yet).

**Comment by wwmm** [comment]
@francoism90 Happy New Year for you too! :-)

>It seems installing pipewire-pulse replaces Pulseaudio, is that correct? 

Yes. `pipewire-pulse` is PipeWire's compatibility layer for Pulseaudio applications. It implements Pulseaudio's protocol in a way that applications "think" that Pulseaudio is still there.

>I'm on KDE and I don't think they do support Pipewire

PipeWire's compatibility layer has evolved to a point where I am able to use mpv, chromium, firefox, gnome, etc without major issues. VLC is still broken but it is possible KDE is working with the compatibility layer. The main problem is that the current PulseEffects package in the repository does not work with this compatibility layer. Or at least wasn't the last time I tried.

I have already started the port to PipeWire and our master branch is using it instead of Pulseaudio. Almost all functionalities are already working. Only the minor ones like preset autoloading and test signals were not done yet.
